src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -66,34 +64,11 @@ protected void textChanged(final DocumentEvent e) {
       }
     });
 
-    final TextComponentAccessor<JComboBox> textComponentAccessor = new TextComponentAccessor<JComboBox>() {
-      @Override
-      public String getText(final JComboBox component) {
-        return component.getEditor().getItem().toString();
-      }
-
-      @Override
-      public void setText(@NotNull final JComboBox component, @NotNull final String text) {
-        if (!text.isEmpty() && !FlutterSdkUtil.isFlutterSdkHome(text)) {
-          final String probablySdkPath = text + "/dart-sdk";
-          if (FlutterSdkUtil.isFlutterSdkHome(probablySdkPath)) {
-            component.getEditor().setItem(FileUtilRt.toSystemDependentName(probablySdkPath));
-            return;
-          }
-        }
-
-        component.getEditor().setItem(FileUtilRt.toSystemDependentName(text));
-        isModified = true;
-      }
-    };
-
+    sdkCombo.addBrowseFolderListener("Select Flutter SDK Path", null, null,
+                                     FileChooserDescriptorFactory.createSingleFolderDescriptor(),
+                                     TextComponentAccessor.STRING_COMBOBOX_WHOLE_TEXT);
 
-    final ComponentWithBrowseButton.BrowseFolderActionListener<JComboBox> browseFolderListener =
-      new ComponentWithBrowseButton.BrowseFolderActionListener<>("Select Flutter SDK Path", null, sdkCombo,
-                                                                 null,
-                                                                 FileChooserDescriptorFactory.createSingleFolderDescriptor(),
-                                                                 textComponentAccessor);
-    sdkCombo.addBrowseFolderListener(null, browseFolderListener);
+    versionDetails.setBackground(UIUtil.getPanelBackground());
```

Comment:
How about showing version details as a standard multiline JLabel? It will look the same as 'Flutter SDK path' or 'Version details' label an will take several lines according to the `flutter -version` output. Happy to update the PR.